cases/operator-crashloop-missing-perms/deployment.yaml

```diff
@@ -1,32 +1,42 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: rbac-test
+  name: nginx
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: rbac-test
+      app: nginx
   template:
     metadata:
       labels:
-        app: rbac-test
+        app: nginx
     spec:
-      serviceAccountName: restricted-sa
+      serviceAccountName: nginx-sa
       containers:
-      - name: kubectl-runner
-        image: bitnami/kubectl:latest
-        command: ["/bin/sh", "-c"]
-        args:
-        - |
-          while true; do
-            kubectl get pods --all-namespaces
-            sleep 10
-          done
-        resources:
-          requests:
-            cpu: 50m
-            memory: 128Mi
-          limits:
-            cpu: 100m
-            memory: 256Mi 
\ No newline at end of file
+        - name: nginx
```

Comment:
this is still not a kubernetes operator, you should be deploying something like cert manager or istio or a related resource.